sarama.go

```diff
@@ -22,6 +24,8 @@ import (
 // but you can set it to redirect wherever you want.
 var Logger StdLogger = log.New(ioutil.Discard, "[Sarama] ", log.LstdFlags)
 
+var Random = rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Should this really be public? I'm not sure why anyone would want to override it. If you want to leave it public, it needs godoc